src/coreComponents/mesh/generators/MeshGeneratorBase.hpp

```diff
@@ -122,12 +129,30 @@ class MeshGeneratorBase : public dataRepository::Group
    */
   std::map< string, string > const & getSurfacicFieldsMapping() const { return m_surfacicFields; }
 
+  /**
+   * @brief Get the associatied SpatialPartition generated.
+   * @return The generated SpatialPartition
+   */
+  SpatialPartition const & getSpatialPartition() const { return m_spatialPartition; }
+private:
+  /**
+   * @brief Fill the cellBlockManager object .
+   * @param[in] cellBlockManager to fill with the mesh informations
+   * @param[in] partition The number of domains in each dimesion (X,Y,Z)
+   */
+  virtual void fillCellBlockManager( CellBlockManager & cellBlockManager, array1d< int > const & partition ) = 0;
+
+  void attachWellInfo( CellBlockManager & cellBlockManager );
```

Comment:
Can you put the `private` below the `protected`?

Reply:
👍